src/renderer/domain/connection.ts

```diff
@@ -9,6 +9,7 @@ export type Connection = {
   customNodes?: RpcNode[];
   activeNode?: RpcNode;
   disconnect?: () => void;
+  hasMultisigPallet?: boolean;
```

Comment:
Does It need to be optional?
I see `hasMultisigPallet: Boolean(api?.tx.multisig)` down below, looks like we have the value all the time

Reply:
Not all the time, we have some time between first app start and first chain connection

Same for when we will add new chains